src/status_im/translations/fr.cljs

```diff
@@ -117,15 +113,13 @@
    :confirmation-code                     (str "Merci ! Nous vous avons envoyé un message texte contenant le code "
                                                "de confirmation. Veuillez fournir ce code pour confirmer votre numéro de téléphone")
    :incorrect-code                        (str "Désolé, le code est incorrect. Veuillez le saisir à nouveau.")
-   :generate-passphrase                   (str "Je vais créer une phrase secrète pour vous permettre de restaurer votre "
-                                               "accès ou de vous connecter depuis un autre périphérique")
    :phew-here-is-your-passphrase          "*Ouf* c'était dur, voici votre phrase secrète, *écrivez-la et gardez-la en lieu sûr !* Vous en aurez besoin pour restaurer votre compte."
    :here-is-your-passphrase               "Voici votre phrase secrète, *écrivez-la et gardez-la en lieu sûr !* Vous en aurez besoin pour restaurer votre compte."
-   :written-down                          "Assurez-vous de l'avoir bien écrite"
+   :here-is-your-signing-phrase           "Voici votre phrase de signature. Vous l'utiliserez pour vérifier vos signatures. *Ecrivez la et conservez la soigneusement!*"
    :phone-number-required                 "Appuyez ici pour entrer votre numéro de téléphone et je vais trouver vos amis"
    :shake-your-phone                      "Vous avez trouvé un bug ou alors avez une suggestion? ~Secouez~ simplement votre téléphone!"
    :intro-status                          "Chattez avec moi pour configurer votre compte et modifier vos paramètres !"
-   :intro-message1                        "Bienvenue dans Status\nTapez sur ce message pour définir votre mot de passe et commencer !"
+   :intro-message1                        "Bienvenue dans Statut\nTapez sur ce message pour définir votre mot de passe et commencer !"
```

Comment:
We don't want to translate `Status`

Reply:
Oh I see, didn't realise it was the brand name :S . I'll put it back.